console_backend/src/ntrip_output.rs

```diff
@@ -1,32 +1,46 @@
-use crate::ntrip_tab::OutputType;
 use crate::types::{ArcBool, Result};
 use anyhow::Context;
 use crossbeam::channel::Receiver;
 use log::error;
 use std::io::Write;
 use std::process::{Command, Stdio};
-use std::{io, thread};
+use std::{fs, io, thread};
 
 pub struct MessageConverter {
     in_rx: Receiver<Vec<u8>>,
     running: ArcBool,
-    output_type: OutputType,
+    binary_path: String,
```

Comment:
This is totally fine, but i do feel like it might be a good usecase for UTF8Path & the camino crate: https://docs.rs/camino/latest/camino/struct.Utf8Path.html